src/extension.ts

```diff
@@ -22,7 +22,7 @@ function createLangServer(context: ExtensionContext): LanguageClient {
     const packageJson = extensions.getExtension('sourcery.sourcery').packageJSON;
     const extensionVersion = packageJson.version;
 
-    const command = path.join(__dirname, "..", "sourcery_binaries/" + getExecutablePath());
+    const command = path.join(__dirname, "..", "..", "core", "run-sourcery.sh");
```

Comment:
We shouldn't commit this. :slightly_smiling_face: 
And perhaps, we can define an env variable instead of always adjusting this manually.